### PR TITLE
DRUP-1040: Prevent clicking active links in sidebar nav

### PR DESCRIPTION
**Pull request by jameswilson**
@akohler this issue partially solves the problems discussed on DRUP-1040 by preventing clicks on the sidebar "active" menu link.  

This can be tested to confirm to work as expected on both desktop and mobile on the following three pages.

https://drup-1040-unclickable-4zxrjqy-sopc3ixpdigl4.us-2.platformsh.site/use
https://drup-1040-unclickable-4zxrjqy-sopc3ixpdigl4.us-2.platformsh.site/use/access-privileges
https://drup-1040-unclickable-4zxrjqy-sopc3ixpdigl4.us-2.platformsh.site/use/access-privileges/privacy-policy

Unfortunately though, when this is approved for push to staging, there is another commit on Master that has not been synced properly with staging, related to DRUP-1247 that should also go out with this fix.

### CODE REVIEW

**Comment by akohler** [comment]
@jameswilson What specific commit for DRUP-1247 needs attention for staging?

**Comment by jameswilson** [comment]
@akohler 9c74261d

**Comment by jameswilson** [comment]
@akohler I've actually modified the part of that commit that unsets the title on sub-sub pages, because it was breaking the collapsible menu for mobile.  

https://github.com/UCLALibrary/libraryweb-site/commit/9c74261dafb5d4d564debb452fd1f5f20db6938a#diff-db32c328f0adcb5f70ff879284810334

This is evident on:

https://drup-1040-unclickable-4zxrjqy-sopc3ixpdigl4.us-2.platformsh.site/use/access-privileges/privacy-policy

If you compare it with http://www-test.library.ucla.edu/use/access-privileges/privacy-policy where the at-a-glance menu dropdown is hidden, and only shows a 4 pixel blue line on ww-test.

File: www/sites/all/modules/features/uclalib_navigation/plugins/ctools/content_types/uclalib_nav.inc [hunk 1]
```diff
@@ -99,13 +99,13 @@ function uclalib_navigation_ctools_content_types_uclalib_nav_render($subtype, $c
   // This is a sub-section or below, the title should link to the parent.
   if (count($menu_trail) > 2) {
     if ($has_children) {
-      $block->title = l(t('At a Glance'), $current['link_path']);
+      $block->title = l(t('At a Glance'), $current['link_path'], array('attributes' => array('class' => array('nav-current-path', 'nav-has-menu-trail'))));
     } else {
-      unset($block->title);
+      $block->title = l(t('At a Glance'), $parent['link_path'], array('attributes' => array('class' => array('nav-parent-path', 'nav-has-menu-trail'))));
```
Comment (jameswilson) [inline www/sites/all/modules/features/uclalib_navigation/plugins/ctools/content_types/uclalib_nav.inc]: This line resets the functionality back to the way it was before commit https://github.com/UCLALibrary/libraryweb-site/commit/9c74261dafb5d4d564debb452fd1f5f20db6938a#diff-db32c328f0adcb5f70ff879284810334  in order to restore the mobile "at a glance" sub-nav dropdown on sub-sub pages like /use/access-privileges/privacy-policy

**Comment by jameswilson** [comment]
ugh. ok.  so it looks like the link was removed due to accessibility alert about a redundant link, so there may be more work that needs to be done here to figure out a compliant (and UX-friendly) proper multi-level navigation system.

**Comment by jameswilson** [comment]
Status here:  my work on this PR will “unbreak” something that is currently broken on master, but since that commit never got pushed live, the accessibility alert is already/still there on production, so we don't technically lose anything by pushing this work live, and we fix the original ticket for which this issue was created.  I think we could then open a follow-up to properly re-address the accessibility issue and possibly the UX issues, given we now have more context about the behavior.

**Pull request merged**